src/hugr/views/snapshots/hugr__hugr__views__tests__mermaid_string.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: src/hugr/views/tests.rs
+assertion_line: 77
+expression: h.mermaid_string()
+---
+graph LR
+    subgraph 0 ["(0) DFG"]
+        direction LR
+        1["(1) Input"]
+        1--"0:0<br>qubit([])"-->3
+        1--"1:1<br>qubit([])"-->3
+        2["(2) Output"]
+        3["(3) test.quantum.CX"]
+        3--"0:1<br>qubit([])"-->4
+        3--"1:0<br>qubit([])"-->4
+        3-."2:2".->4
+        4["(4) test.quantum.CX"]
+        4--"0:0<br>qubit([])"-->2
+        4--"1:1<br>qubit([])"-->2
```

Comment:
I don't think this tests all the possibles very well, maybe a more complicated test involving nested graphs, non-local static edges and a cfg?